### PR TITLE
po: regular (minor) updates

### PR DESCRIPTION
**Pull request by dmedri**
-it.po (italian)
- systemd.pot (updated)

### CODE REVIEW

File: po/it.po [hunk 1]
```diff
@@ -2,21 +2,21 @@
...
 # Traduzione in italiano per il pacchetto systemd
-# Daniele Medri <dmedri@gmail.com>, 2013-2020.
+# Daniele Medri <dmedri@gmail.com>, 2013-2021.
 msgid ""
 msgstr ""
 "Project-Id-Version: systemd\n"
-"Report-Msgid-Bugs-To: https://github.com/systemd/systemd/issues\n"
-"POT-Creation-Date: 2020-08-19 18:02+0200\n"
-"PO-Revision-Date: 2020-10-03 09:52+0200\n"
+"Report-Msgid-Bugs-To: \n"
```
Comment (bluca) [inline po/it.po:9]: Why was the URL removed?

File: po/it.po [hunk 2]
```diff
@@ -311,279 +314,295 @@ msgstr ""
 "gestione di sistema alla apertura/chiusura del portatile."
 
 #: src/login/org.freedesktop.login1.policy:117
+msgid "Allow applications to inhibit system handling of the reboot key"
+msgstr ""
+"Consenti alle applicazioni di inibire la gestione di sistema del tasto di "
```
Comment (bluca) [inline po/it.po:319]: "la gestione di sistema" doesn't sound great, I think this is better:

```
Consenti alle applicazioni di impedire al sistema di gestire il tasto di riavvio
```

File: po/it.po [hunk 3]
```diff
@@ -311,279 +314,295 @@ msgstr ""
...
+"riavvio"
+
+#: src/login/org.freedesktop.login1.policy:118
+msgid ""
+"Authentication is required for an application to inhibit system handling of "
+"the reboot key."
+msgstr ""
+"Autenticazione richiesta affinché un'applicazione possa inibire la gestione "
```
Comment (bluca) [inline po/it.po:327]: Same:

```
Autenticazione richiesta affinché un'applicazione possa impedire al sistema di gestire il tasto di riavvio.
```

File: docs/TRANSLATORS.md [hunk 4]
```diff
@@ -26,7 +26,7 @@ To create a translation to a language not yet available, start by creating the
 initial template:
 
 ```
-$ meson compile -C build/ systemd-pot
+$ ninja -C build/ systemd-pot
```
Comment (bluca) [inline docs/TRANSLATORS.md:29]: these were changed intentionally in https://github.com/systemd/systemd/pull/17939 - any specific reason to switch back? If not, please drop this commit

Comment (dmedri) [reply docs/TRANSLATORS.md:29]: For me, it doesn't run the old version. I'll remove this commit, but a check is needed.

File: po/it.po (hunk 3 above)
Comment (dmedri) [reply po/it.po:327]: Nice tip, I'll review even other labels.

File: po/it.po (hunk 1 above)
Comment (dmedri) [reply po/it.po:9]: Incidental poedit removal. I'll restore it. Thank you

**Pull request closed without merging**